src/lib/mongoBackend/safeBsonGet.cpp

```diff
@@ -105,8 +102,31 @@ int getIntField(const BSONObj& b, const std::string& field)
 
 /* ****************************************************************************
 *
-* getBoolField -
+* getLongField -
+*/
+long getLongField(const BSONObj& b, const std::string& field)
```

Comment:
Sure, but if we use Long always, the risk disappears, right?
The price to pay is to get the databases a little bigger.
To me, this is the best option, risk free, fast an a little bigger.

We should create an issue about this. The decision can be made later